Content.Client/PDA/Ringer/RingtoneMenu.xaml.cs

```diff
@@ -0,0 +1,105 @@
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Maths;
+using Content.Client;
+
+namespace Content.Client.PDA.Ringer
+{
+    [GenerateTypedNameReferences]
+    public partial class RingtoneMenu : SS14Window
+    {
+        private string[]? previousNoteInputs = new string[4];
```

Comment:
Please look at https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions around this and your other fields. It should be prefixed by a _.